include/lemlib/chassis/differential.hpp

```diff
@@ -132,7 +134,7 @@ struct ControllerSettings {
  * @param rightMotors pointer to the right motors
  * @param trackWidth the track width of the robot
  * @param wheelDiameter the diameter of the wheel used on the drivetrain
- * @param rpm the rpm of the wheels
+ * @param speed the rpm of the wheels
```

Comment:
```suggestion
 * @param speed the angular velocity of the wheels (e.g 360_rpm)
```
Update doxygen comment block